### DISCUSSION TITLE
How to resize node size?

### DISCUSSION THREAD

**Discussion by leenzhu** [WerWolv/ImHex#1644] [Get Help] [answered]
Hi,

I'm using a small screen with resolution 2560 x 1600 and scale with 200%，and I config the ImHex with sacle 2.x too.

everything looks well as expected but when I add a buffer display node, it looks too large:
![image](https://github.com/WerWolv/ImHex/assets/25599632/786277d5-2d26-4e9e-bbfd-b5ec98c00ed2)

How to resize the node size?

**Comment by WerWolv** [comment, marked as answer]
Hey, this was a bug. Latest commit should fix it :)
